tests/FSharp.Compiler.Tests/Tests.fs

```diff
@@ -192,7 +192,7 @@ module TestModuleCompilationTest =
         let speculativeSymbolInfo = semanticModel.GetSpeculativeSymbolInfo (position, node, CancellationToken.None)
 
         Assert.True (symbol.IsSome)
-        Assert.False (speculativeSymbolInfo.GetAllSymbols().IsEmpty)
```

Comment:
If possible we should avoid Assert.True and Assert.False and go for more specific versions. Especially if we have multiple assertions this is recipe for having a hard time once the test fails and we need to discover why. If we can't use a more specific assertion then we should at least use the overload that takes a string as additional message.